src/routes/form-title.tsx

```diff
@@ -18,11 +18,12 @@ const TitleContainer = styled.div`
 `;
 
 const Title = styled(Innholdstittel)`
-    @media ${device.mobileS} {
-        margin-top: 0;
-    }
+    margin-bottom: 0;
+
     @media ${device.mobileL} {
-        margin-top: 8px;
+        && {
+            margin-bottom: 8px;
```

Comment:
Har du byttet på `margin-top` og `margin-bottom` med vilje her?

Reply:
Ja, denne marginen styrer avstanden mellom tittel og undertittel.
Den var nok `margin-top` for undertittel tidligere.